tests/test_presentation.py

```diff
@@ -136,6 +145,7 @@ def test_existing_event_identifier_results_in_HttpResponse(self, event_xml):
 
         assert isinstance(event_or_response, HttpResponse)
 
+    @pytest.mark.xfail(reason='For consistency\'s sake, all duplicate IDs should 409.')
```

Comment:
Rather than leave this marked as a test we expect to fail, can we change it to a test that we expect to pass with our now consistent behavior?